examples/skr/aks/README.md

```diff
@@ -0,0 +1,228 @@
+The release policy describes the conditions the key release request must meet in order for the mhsm to release the key.
+
+To set up a mHSM instance, follow the instructions [here with Azure CLI](https://learn.microsoft.com/en-us/azure/key-vault/managed-hsm/quick-create-cli),
+or through the [Azure portal](https://ms.portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.KeyVault%2FmanagedHSMs).
+You can follow the installation instructions for Azure CLI [here](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli).
+
+### 2. Obtain an Attestation Endpoint
+
+Users must have a working attestation endpoint to interact with the mHSM.
+
+If you don't already have a valid attestation endpoint, create a [Microsoft Azure Attestation](https://learn.microsoft.com/en-us/azure/attestation/overview) endpoint to author the attestation token and run the following command to get the endpoint value:
+
+```shell
+az attestation show --name "<ATTESTATION PROVIDER NAME>" --resource-group "<RESOURCE GROUP>"
+```
+
+### 3. Generate a Key Pair
+
+This is a one time effort.
+Once the key is created, it can be used to protect as many secrets as desired.
+
+The `setup-key-mhsm.sh` script creates a private/public key pair in the keyvault and a key release policy with the file name `keyname-release-policy.json`.
+The public key is then downloaded as `keyname-pub.pem` and a key info file is generated.
+Both the public key and the key info file are used for wrapping the secret.
+The script also assigns the necessary access role (Managed HSM Crypto User) to the Managed Identity.
+Depending on the operating system, make sure the end of line sequence(LF vs CRLF) is set correctly before running the script.
+
+`MANAGED_IDENTITY` has the following format:
+
+`/subscriptions/<subscription-id>/resourceGroups/<resource_group_name>/providers/Microsoft.ManagedIdentity/<userAssignedIdentities>/msi`
+
+`<key-name>`: just the name of the key such as `examplekey`
+
+`<mhsm>`: just the name of the mhsm instance without the `managedhsm.azure.net` part.
+For a mhsm instance with the full URL: `examplemhsm.managedhsm.azure.net`, the `mhsm` is just `examplemhsm`.
+
+```bash
+# Create a managed identity for accessing MHSM. Note the Principle ID of the identity
+az identity create -g $RESOURCE_GROUP -n $USER_ASSIGNED_IDENTITY_NAME | grep "id"
+# The following two exported env vars are used by the setup-key-mhsm.sh script
+export MANAGED_IDENTITY=<principle-id>
+export MAA_ENDPOINT=<maa-endpoint> # Choose a MAA instance for the attestation service, e.g. sharedeus2.eus2.attest.azure.net
+
+# Login
+az login
+# Set account context to the subscription where the mhsm resides
+az account set --subscription <subscription-name>
+# change the access permissions to execute of the script
+chmod +x setup-key-mhsm.sh
+# execute the script to generate key pair
+bash setup-key-mhsm.sh <key-name> <mhsm>
```

Comment:
Instead of providing the mhsm name as an input, we now provide the full url of the akv/mhsm.

Reply:
If it's not in this repo, we're not mentioning it. We can do a future PR to update things to match what's going on in the Kafka demo.